vfs/rc.go

```diff
@@ -86,9 +86,33 @@ starting with dir will forget that dir, eg
 				return nil, EINVAL
 			}
 
+			recursive := false
+			{
+				const k = "recursive"
+
+				if v, ok := in[k]; ok {
+					s, ok := v.(string)
+					if !ok {
+						return out, errors.Errorf("value must be string %q=%v", k, v)
+					}
+					switch strings.ToLower(s) {
```

Comment:
You can use `strconv.ParseBool` to parse this.

Reply:
Good point. I forgot that `strconv.ParseBool` will parse all that variations.